templates/taxi/manufacturer_form.html

```diff
@@ -9,4 +9,4 @@ <h1>{{ object|yesno:"Update,Create" }} manufacturer</h1>
 
     <input type="submit" value="Submit" class="btn btn-primary">
   </form>
```

Comment:
According to the code style point in the checklist, it is recommended to add a `Cancel` button that will lead to the previous page the user was on. Consider adding this functionality to improve user experience.